tests/test_simulation.py

```diff
@@ -60,14 +60,22 @@ def testSimulationEigen(self):
         self.assertTrue(np.allclose(
             eigs_true[:, :, 2, 0],
             np.array([
-                [-1.67666201e-07, -7.95741380e-06, -1.49160041e-04, -1.10151654e-03, -3.11287888e-03, -3.09157884e-03, -9.91418026e-04, -1.31673165e-04],
-                [-1.15402077e-06, -2.49849709e-05, -3.51658906e-04, -2.21575261e-03, -7.83315487e-03, -9.44795180e-03, -4.07636259e-03, -9.02186439e-04],
-                [-1.88737249e-05, -1.91418396e-04, -1.09021540e-03, -1.02020288e-03,  1.39411855e-02,  8.58035963e-03, -5.54619730e-03, -3.86377703e-03],
-                [-1.21280536e-04, -9.51461843e-04, -3.22565017e-03, -1.05731178e-03,  2.61375736e-02,  3.11595201e-02,  6.40814053e-03, -2.31698658e-02],
-                [-2.44067283e-04, -1.40560151e-03, -6.73082832e-05,  1.44160679e-02,  2.99893934e-02,  5.92632964e-02,  7.75623545e-02,  3.06570008e-02],
-                [-1.53507499e-04, -7.21709803e-04,  8.54929152e-04, -1.27235036e-02, -5.34382043e-03,  2.18879692e-02,  6.22706190e-02,  4.51998860e-02],
-                [-3.00595184e-05, -1.43038429e-04, -2.15870258e-03, -9.99002904e-02, -7.79077187e-02, -1.53395887e-02,  1.88777559e-02,  1.68759506e-02],
-                [ 3.22692649e-05,  4.07977635e-03,  1.63959339e-02, -8.68835449e-02, -7.86240026e-02, -1.75694861e-02,  3.24984640e-03,  1.95389288e-03]
+                [-1.67666201e-07, -7.95741380e-06, -1.49160041e-04, -1.10151654e-03,
```

Comment:
Why do these have to be changed?

Reply:
This is to make the representation of covariance more accurate. The related commit has been reverted and make it be addressed in more details in future issue.